lib/tasks/generate.rake

```diff
@@ -0,0 +1,7 @@
+namespace :generate do
```

Comment:
You can wrap this task definition in something like `unless Rails.env.production?` to ensure we could never run these tasks in a prod environment.

Reply:
We had these checks in the inner-layers, but also aggressively added checks to all boundary layers.